release/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM ubuntu:20.04
+
+COPY config /config
+COPY target/aarch64-unknown-linux-gnu/debug/matchengine /usr/bin/
```

Comment:
why aarch64 rather than x64/amd64?

aarch64 is arm64/armv8

Reply:
Needs to be fixed to `x86_64-unknown-linux-gnu`.